css/main.css

```diff
@@ -0,0 +1,408 @@
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.contact,
+.contact_wrap,
+.contact_form {
+  display: flex;
+}
+
+.contact_wrap,
+.contact_form {
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.contact {
+  background: #6070ff;
+  border-top-left-radius: 100px;
+  height: 110vh;
+  min-height: 900px;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact_wrap {
+  min-width: 50vw;
+  height: 70vh;
+  margin-top: 15vh;
+  align-items: center;
+}
+
+.contact_wrap,
+.textarea {
+  font-family: 'Poppins', sans-serif;
+}
+
+.head {
+  color: #fff;
+  font-size: 2rem;
+  margin-bottom: 3vh;
+}
+
+.bodyText {
+  color: #ebebff;
+  font-size: 18px;
+}
+
+.contact_form {
+  margin: 5vh 0 3vh 0;
+}
+
+.form_fields,
+.bodyText {
+  width: 40vw;
+  min-width: 220px;
+}
+
+.form_fields {
+  height: 7vh;
+  margin-bottom: 3vh;
+  padding: 10px;
+  font-size: 17px;
+}
```

Comment:
- [ ] As it has already been pointed out, it's best to follow the given Figma design. While doing so, please use CSS flexbox to style the contents of your form.